src/Razor/src/Microsoft.VisualStudio.RazorExtension/RazorPackage.cs

```diff
@@ -69,6 +69,10 @@ protected override async Task InitializeAsync(CancellationToken cancellationToke
                 var toolwndCommandID = new CommandID(GuidSyntaxVisualizerMenuCmdSet, (int)CmdIDRazorSyntaxVisualizer);
                 var menuToolWin = new MenuCommand(ShowToolWindow, toolwndCommandID);
                 mcs.AddCommand(menuToolWin);
+
+                // Initialize command handlers in the window
+                var window = (SyntaxVisualizerToolWindow)FindToolWindow(typeof(SyntaxVisualizerToolWindow), id: 0, create: true);
```

Comment:
This is gonna regress RPS isn't it 😛

Reply:
Oooh, maybe. I can move to lazy initialize on first showing